src/handlers/options.ts

```diff
@@ -12,17 +12,11 @@ export function getOptionHandler(dbPool: PostgresJsDatabase<typeof db>) {
       return res.status(400).json({ error: 'Missing optionId' });
     }
 
+    const { voteScore, ...rest } = getTableColumns(db.questionOptions);
+
     const rows = await dbPool
       .select({
-        id: db.questionOptions.id,
-        userId: db.questionOptions.userId,
-        registrationId: db.questionOptions.registrationId,
-        questionId: db.questionOptions.questionId,
-        optionTitle: db.questionOptions.optionTitle,
-        optionSubTitle: db.questionOptions.optionSubTitle,
-        accepted: db.questionOptions.accepted,
-        createdAt: db.questionOptions.createdAt,
-        updatedAt: db.questionOptions.updatedAt,
+        ...rest,
```

Comment:
Vote score should be excluded as part of the option handler.

Reply:
I believe it is being excluded